tests/omhttp-basic.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# This file is part of the rsyslog project, released under ASL 2.0
+port=8080
+omhttp_start_server -p $port
+
+#  Starting actual testbench
+. ${srcdir:=.}/diag.sh init
+
+error_file=$(pwd)/rstb.omhttp.error.log
```

Comment:
We cannot use fixed path, because test can (and do) run in parallel.

So this needs to be
```
error_file=$RSYSLOG_DYNNAME.omhttp.error.log
```